grouper/fe/handlers.py

```diff
@@ -821,6 +821,12 @@ def post(self, audit_id):
             return self.forbidden()
 
         audit = self.session.query(Audit).filter(Audit.id == audit_id).one()
+
+        # only owners can complete
+        owner_ids = {member.id for n,member in audit.group.my_owners().items()}
```

Comment:
This seems a bit odd to me, since I'm immediately wondering what the values of that dict are (and `n` is unused).  How about:

```
owner_ids = {member.id: True for _, member in audit.group.my_owners().items()}
```